pages/how-to/installation.md

```diff
@@ -1,7 +1,31 @@
 # Installing fpm
 
-This how-to guide covers the installation of the Fortran package manager (fpm) on various platforms.
+This how-to guide covers the installation of the Fortran Package Manager (fpm) on various platforms.
```

Comment:
I would keep this lowercase

Reply:
This is actually a general question since it should be consistent for all documents. Maybe we can do this change separately rather than in three different patches? This way we can also adapt the translation files in the same patch.

Reply:
We can do it in separate patches.

The choice boils down to whether we mean "Fortran Package Manager" as proper noun (I think so, and it allows abbreviating to fpm), in which case the correct spelling is to capitalize each word.

Reply:
Let's make a separate patch and replace it in all files, maybe an automatic replacement in all files will work sufficiently well (also capturing the msgid's in the po files).

Reply:
I agree with Milan, we're referring to fpm as a proper noun and hence it should be capitalized as such.